docs/directory-structure.md

```diff
@@ -120,15 +120,10 @@ Django context (i.e. with database access).
 
 ### API and Bots
 
-* `api/` Zulip's Python API bindings (released separately).
-
-* `api/examples/` API examples.
-
-* `api/integrations/` Integrations distributed as part of the Zulip API bundle.
-
-* `api/bots/` Bots distributed as part of the Zulip API bundle.
-
-* `api/bots_api/` Framework for running and testing bots in `api/bots/`.
+See the [Zulip API repository](https://github.com/zulip/python-zulip-api).
+Zulip's Python API bindings, a number of Zulip integrations and bots,
+and a framework for running and testing Zulip bots, used to be developed
+in the main Zulip server repo but are now in their own repo.
```

Comment:
Might make sense to leave behind a bullet for `templates/zerver/integrations/`, which is where the docs for the integrations included in the `zulip` Python package live.